### DISCUSSION TITLE
Desktop-Browser authentification for "publish repository" : User interaction failed error

### DISCUSSION THREAD

**Discussion by Patacloko** [community/community#65592] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Hey everyone, hope you're doing fine. 

I tried to connect my Desktop Github ( Version 3.2.9-linux1 (x64) ) to my account so to publish my repository, only to be met by an error message : "user interaction failed". 

Here is the image : 
![Error git](https://github.com/community/community/assets/141639333/b1ebfcf1-47fa-41c9-b545-8152451d63f0)

To sign in the Desktop app, the identification must be made on a browser. I've tried connecting with Brave Browser and Firefox, I got the same results with both. 

Now this is very annoying because i really like Linux, and really like Github, and i don't want to have to chose one over the other. It seems that other discussions with people that have faced the same error have been opened before but ended up quickly closed, due to "Github only being officially supported on Mac and Windows".  I have found no answer or fix trough browsing those discussions. I'm making this post because it seems not to be an isolated problem. 

Replicate my method : Create repository in folder / Publish repository / Sign in / Connect in browser / Allow Github Desktop to be opened by browser / error : user interaction failed 

System : 
Nobara Linux 36
Kde Plasma 5.27.4

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Patacloko ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by EstebanMagallonPerez** [comment]
This worked for me to login: 
In flatseal go to the chrome permissions and enable "D-bus session bus" this will allow you to authenticate.

**Reply by LazyGamesDev** [reply]
This worked for me - Also try this alternative : 
Flatpak Permissions > Your browser [ Firefox / Chrome / etc ] > Advanced > Enable Session bus access & System Bus Access

**Comment by Shubrathshetty** [comment]
Run these commands one after the other:
sudo dnf install gnome-keyring

eval $(gnome-keyring-daemon --start --components=secrets)
export SSH_AUTH_SOCK

rm -rf ~/.config/GitHub\ Desktop/

After the setup relaunch GitHub Desktop and sign in normally.
Hope this works...
